contrib/config

```diff
@@ -0,0 +1,7 @@
+music-path        
+cache-path        /tmp/gonic_cache
+db-path           var/gonic/gonic.db
```

Comment:
should this instead be `/var/gonic/gonic.db`?

Reply:
Oh yes, your right.